packages/helloworld/scripts/android.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
not sure if we should land it, we can but this will super temporary and tbh I would just properly implement `run:android` and `run:ios` commands